asistente_ladm_col/logic/ladm_col/data/ladm_data.py

```diff
@@ -203,19 +460,21 @@ def get_parcel_data_to_compare_changes(self, db, search_criterion=None, layer_mo
             dict_attrs = dict()
             for field in layers[self.names.OP_PARCEL_T][LAYER].fields():
                 if field.name() in parcel_fields_to_compare:
+                    key_value_parcel = mapping_parcels_field[field.name()] if field.name() in mapping_parcels_field else field.name()
```

Comment:
Create a new function (e.g., `get_key_for_field()`) that resolves keys logic out of the current function.
Try to use the dict.keys() as the `parcel_fields_to_compare` list to avoid the latter list.

Reply:
Adjustment was made, thanks.